content/blog/2020/08/2020-08-04-cdf-graduation.adoc

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: "Jenkins graduates in the Continuous Delivery Foundation"
+tags:
+- announcement
+- cdf
+- general
+- community
+author: oleg_nenashev
+description: >
+  The Jenkins project has graduated in the Continuous Delivery Foundation (CDF).
+  In this article, we discuss what it means to the Jenkins community and what are the future steps for the project.
+opengraph:
+  image: /images/post-images/2020/08-cdf-graduation/opengraph.png
+---
+
+We are happy to announce that the Jenkins project has achieved the **graduated status** in
+the link:https://cd.foundation/[Continuous Delivery Foundation (CDF)].
+This status is official effective Aug 03, 2020.
+Jenkins is the first project to graduate in the CD Foundation.
```

Comment:
Some places are using "CD Foundation" and others are using "CDF", I would recommend using one or the other.

Reply:
I mostly operated based on the line breaks and line alignment. Can align everything to the CD Foundation to follow the official announcamants

Reply:
I have patched a few instances, but even the CDF materials use both ways. If you are fine with that, I would leave it as is